This patch is contains the following upstream commits:
	* b094e11e24e5fb7672d73613cceb4af4cd8158bc
		set cmake_min_req to enable newer policies

As well as the FreeBSD specific fix from
	* r288120
		Work around the "automoc hangs forever while building kde" problem.


--- CMakeLists.txt.orig	2009-01-22 18:50:09 UTC
+++ CMakeLists.txt
@@ -1,7 +1,6 @@
+cmake_minimum_required(VERSION 2.8.9)
 project(Automoc4)

-cmake_minimum_required(VERSION 2.4.5 FATAL_ERROR)
-
 find_package(Qt4 REQUIRED)

 if (CMAKE_COMPILER_IS_GNUCXX OR CMAKE_C_COMPILER MATCHES "icc")
@@ -21,7 +20,9 @@ add_executable(automoc4 kde4automoc.cpp)
 set_target_properties(automoc4  PROPERTIES  SKIP_BUILD_RPATH            FALSE
                                             INSTALL_RPATH_USE_LINK_PATH TRUE )

-target_link_libraries(automoc4 ${QT_QTCORE_LIBRARY})
+find_package(Threads)
+target_link_libraries(automoc4 ${QT_QTCORE_LIBRARY} ${CMAKE_THREAD_LIBS_INIT})
+

 install(TARGETS automoc4 DESTINATION bin)
 install(FILES Automoc4Config.cmake Automoc4Version.cmake automoc4.files.in  DESTINATION  lib${LIB_SUFFIX}/automoc4)
